astroquery/alfalfa/core.py

```diff
@@ -9,7 +9,7 @@
 import requests
 import numpy as np
 import numpy.ma as ma
-from astropy import units as u
```

Comment:
keep these, as this is the standard way to use astropy.units - it wouldn't be astronomy is there wouldn't be an unlogical contradiction somewhere.

Sorry about that.

Reply:
🙏 Done in 3c5c8fa4afcae93e0f1f4573260ecae7eb45ec43